### PR TITLE
hardening/365_hide_configuration_parameters_in_doc

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue https://github.com/telefonicaid/fiware-cygnus/issues/365 
* No tests are required
* Assignee @frbattid

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -27,3 +27,4 @@
 - [HARDENING] Add cygnus_translator_pre0.10.0_to_0.10.0_mysql_table.sh and fix cygnus_translator_pre0.10.0_to_0.10.0_mysql_db.sh(#765)
 - [HARDENING] User per batch TTL, not per event TTL (#714)
 - [FEATURE] Add enable_lowercase parameter to all sinks (#815)
+- [HARDENING] Comment default configuration parameters in documentation (#365)
```
Comment (frbattid) [inline CHANGES_NEXT_RELEASE]: in documentation and configuration template

File: conf/agent.conf.template [hunk 2]
```diff
@@ -177,13 +185,13 @@ cygnusagent.sinks.postgresql-sink.channel = postgresql-channel
 # sink class, must not be changed
 cygnusagent.sinks.postgresql-sink.type = com.telefonica.iot.cygnus.sinks.OrionPostgreSQLSink
 # true if the grouping feature is enabled for this sink, false otherwise
-cygnusagent.sinks.postgresql-sink.enable_grouping = false
+# cygnusagent.sinks.postgresql-sink.enable_grouping = false
 # true if lower case is wanted to forced in all the element names, false otherwise
-cygnusagent.sinks.hdfs-sink.enable_lowercase = false
+# cygnusagent.sinks.postgresql-sink.enable_lowercase = false
```
Comment (frbattid) [inline conf/agent.conf.template]: Nice catch! :) NTC

File: conf/agent.conf.template [hunk 3]
```diff
@@ -191,15 +199,15 @@ cygnusagent.sinks.postgresql-sink.postgresql_username = root
 # password for the user above
 cygnusagent.sinks.postgresql-sink.postgresql_password = xxxxxxxxxxxxx
 # how the attributes are stored, either per row either per column (row, column)
-cygnusagent.sinks.postgresql-sink.attr_persistence = column
-# select the table type
-cygnusagent.sinks.postgresql-sink.data_model = dm-by-service-path
+# cygnusagent.sinks.postgresql-sink.attr_persistence = row
+# select the data model
```
Comment (frbattid) [inline conf/agent.conf.template]: This comment must be the same than the used with MySQL:

# Must be dm-by-service-path or dm-by-entity

File: conf/agent.conf.template [hunk 4]
```diff
@@ -208,31 +216,37 @@ cygnusagent.sinks.mongo-sink.type = com.telefonica.iot.cygnus.sinks.OrionMongoSi
 # channel name from where to read notification events
 cygnusagent.sinks.mongo-sink.channel = mongo-channel
 # true if the grouping feature is enabled for this sink, false otherwise
-cygnusagent.sinks.mongo-sink.enable_grouping = false
+# cygnusagent.sinks.mongo-sink.enable_grouping = false
 # true if lower case is wanted to forced in all the element names, false otherwise
-cygnusagent.sinks.hdfs-sink.enable_lowercase = false
+# cygnusagent.sinks.mongo-sink.enable_lowercase = false
```
Comment (frbattid) [inline conf/agent.conf.template]: Nice catch! :) NTC

**Comment by frbattid** [comment]
Nice job, a lot of good catches :) Only some minor comments.

Nevertheless, I'm missing some checks regarding the REST handler.

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (pcoello25) [inline CHANGES_NEXT_RELEASE]: Fixed in f982a98

File: conf/agent.conf.template (hunk 3 above)
Comment (pcoello25) [inline conf/agent.conf.template]: Fixed in f982a98

**Comment by pcoello25** [comment]
I've modified the defaults ones in OrionRestHandler as you said :) (in f982a98)

**Comment by frbattid** [comment]
LGTM

**Pull request merged**